tests/interfaces/test_autograd.py

```diff
@@ -482,6 +481,33 @@ class TestParameterHandlingIntegration:
     """Test that the parameter handling for differentiable/non-differentiable
     parameters works correctly."""
 
+    def test_no_differentiable_parameter(self):
+        """Test that the QNode can still be evaluated even when no parameters
+        are differentiable"""
```

Comment:
was this not the case before?

Reply:
It was, but I thought it important we have a test case for it.